CMakeLists.txt

```diff
@@ -29,12 +29,17 @@ endif()
 
 message(STATUS "CMAKE_INSTALL_PREFIX is '${CMAKE_INSTALL_PREFIX}'")
 
+install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/cmake/preproject.cmake
+        DESTINATION cmake
+)
+
 include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/common.cmake)
 
 option(BUILD_TESTS "Build tests" ON)
 option(BUILD_SMALLBANK "Build SmallBank sample app and clients" ON)
 
-# TODO: remove once FTCP support is disabled MemberClient executable
+# MemberClient executable
+# TODO(ftcp): remove once FTCP support is disabled.
```

Comment:
`cmake-format` _really_ wants to reflow our `# TODO`s. We need `# TODO(tag)`, and I haven't found anyway to stop it reflowing trailing lines (only preceding ones).

Reply:
Do we need #TODO(tag)? That's a remnant of the todo tool we used and no longer do. I think we can let cmake-format reflow the comments and get rid of the tags?

Reply:
`cmake-format` also uses these tags. If we have a TODO without the tags

```
# Some permanent comment
# TODO: Important work
```

then this will be reformatted to
```
# Some permanent comment TODO: Important work
```

This garbles multi-line comments or TODOs.

Reply:
Oh ok!